index.html

```diff
@@ -178,6 +178,58 @@ <h2 class="attribute-name">Skills</h2>
         </ul>
     </div>
 </section>
+<section class="contact-section">
+    <div class="contact-header">
+        <h2>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+    </div>
+    <div class="contact-form-container">
+        <form action="https://formspree.io/f/mrgvjkeo" method="post" class="contact-form">
+            
+            <input name="name" type="text" class="full-name" placeholder="Full name" maxlength="30" required>
+            <input name="email-address" type="email" class="email-address" placeholder="Email address" required>
+            <textarea name="message" class="message" placeholder="Hey I would like to connect to..." maxlength="500" required></textarea>
+            <input type="submit" value="Get in touch">
+            
+        </form>
+    </div>
```

Comment:
Please consider making the `width, height, border, etc` properties like the template design used to make it look alike to the design.

### Your design:-

![image](https://user-images.githubusercontent.com/94223367/235961892-dcb054f0-3923-42b6-b577-25102f02fa53.png)


### Figma design:-

![image](https://user-images.githubusercontent.com/94223367/235961928-5e2b3dc6-d434-41e4-abb8-0b2034164b86.png)